### ISSUE TITLE
Unable to load scibert-scivocab-uncased

### ISSUE THREAD

**Issue by vibhas09** [explosion/spacy-transformers#187]
The method given in the README doesn't seem to work for me.

```
from spacy_transformers import TransformersLanguage, TransformersWordPiecer, TransformersTok2Vec
name = "scibert_scivocab_uncased"
path = "scibert/scibert_scivocab_uncased"

nlp = TransformersLanguage(trf_name=name, meta={"lang": "en"})
nlp.add_pipe(nlp.create_pipe("sentencizer"))
nlp.add_pipe(TransformersWordPiecer.from_pretrained(nlp.vocab, path))
nlp.add_pipe(TransformersTok2Vec.from_pretrained(nlp.vocab, path)) 
```

It throws an error:
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
<ipython-input-73-b6f9a932db23> in <module>()
      9 nlp = TransformersLanguage(trf_name=name, meta={"lang": "en"})
     10 nlp.add_pipe(nlp.create_pipe("sentencizer"))
---> 11 nlp.add_pipe(TransformersWordPiecer.from_pretrained(nlp.vocab, path))
     12 nlp.add_pipe(TransformersTok2Vec.from_pretrained(nlp.vocab, path))
     13 

1 frames
/usr/local/lib/python3.6/dist-packages/spacy_transformers/pipeline/wordpiecer.py in from_pretrained(cls, vocab, trf_name, **cfg)
     23     @classmethod
     24     def from_pretrained(cls, vocab, trf_name, **cfg):
---> 25         model = get_tokenizer(trf_name).from_pretrained(trf_name)
     26         return cls(vocab, model=model, trf_name=trf_name, **cfg)
     27 

/usr/local/lib/python3.6/dist-packages/spacy_transformers/util.py in get_tokenizer(name)
    120         return _tokenizers.SerializableXLMTokenizer
    121     else:
--> 122         raise ValueError(f"Unsupported transformers config name: '{name}'")
    123 
    124 

ValueError: Unsupported transformers config name: 'scibert/scibert_scivocab_uncased'
```
My working directory is like this:
- scibert
  - scibert_scivocab_uncased
    - config.json
    - pytorch_model.bin
    - vocab.txt

Package Versions:
- spacy                    2.2.4          
- spacy-transformers       0.5.2

**Comment by adrianeboyd** [comment]
The upcoming 0.6.x will be able to support the following:

```
python examples/init_model.py -n "allenai/scibert_scivocab_uncased" /path/to/scibert_scivocab_uncased_lg
```

If you install `spacy-transformers` from source from `master`, first changing the spacy dependency from `2.3.0` back to `2.2.4` in `setup.cfg`, you should be able to try this out.

(Be aware that the `en_trf_...` models provided by spacy won't be compatible. You can initialize any of your own models with this script, also any of the same ones that would otherwise be available as packaged spacy models. The spacy version change is more to deal with technical problems related to managing the the model incompatibilities rather than any `spacy` features used in `spacy-transformers`.)

**Comment by adrianeboyd** [comment]
This should now work with the newly released `spacy-transformers` v0.6.1.
